src/WorkerChannel.ts

```diff
@@ -167,6 +168,8 @@ export class WorkerChannel implements IWorkerChannel {
                 err = exception;
             }
 
+            this.logColdStartWarning();

```

Comment:
I believe by putting this in `functionLoadRequest`, the log will get written once per function. However, we really only need to log it once per app. With that in mind, I think `workerInitRequest` might be a better place

Reply:
yeah, you are right. moved to `workerInitRequest`